observability/cockpit/concepts.mdx

```diff
@@ -0,0 +1,174 @@
+---
+meta:
+  title: Observability - Concepts
+  description: This page explains all the concepts related to Scaleway Observability
+content:
+  h1: Observability - Concepts
+  paragraph: This page explains all the concepts related to Scaleway Observability
+tags: advanced settings
+categories:
+  - observability
+---
+
+Observability allows you to monitor your applications and their infrastructure by giving you insights and context into their behavior. Observability also enables you to visualize your metrics and logs through a Grafana dashboard.
+
+<Concept opened>
+
+## Alerting
+
+Alerting keeps you aware of issues in your environments, by detecting complex conditions defined by a rule. When a condition defined by a rule is met, the rule tracks it as an alert and responds by triggering one or more actions.
+
+</Concept>
+
+<Concept opened>
+
+## Alert manager
+
+Scaleway's alert manager allows you to manage and respond to alerts. It handles alerts sent when the alerting rules we run are firing. The alert manager triggers alerts (e.g. emails or texts) if a criteria you have configured on your applications' metrics and logs is activated.
+
+</Concept>
+
+<Concept>
+
+## Alerting rules
+
+Alerting rules allow you to define criteria that determine whether an alert will fire. The rule consists of queries and expressions, a condition, the frequency of evaluation, and the duration over which the condition is met. When an alert fires, a notification is sent to the alert manager, which forwards the notification to receivers.
+
+</Concept>
+
+<Concept opened>
+
+## Cockpit
+
+A Cockpit is an Instance of the Observability product that has a dedicated Grafana instance. A Scaleway Project can have only one Cockpit.
```

Comment:
```suggestion
A Cockpit is an instance of the Observability product that has a dedicated Grafana instance. A Scaleway Project can have only one Cockpit.
```

Reply:
In this case when we say "an instance" I think we're definitely not referencing the product